examples/js/postprocessing/DepthPass.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @author jjxtra / https://www.digitalruby.com
+ */
+
+THREE.DepthPass = function(scene, camera)
```

Comment:
You can paste your code into https://zz85.github.io/mrdoobapproves/ to learn how to follow the three.js coding style.

Reply:
Formatted...